_dev/src/views/product-feed-page.vue

```diff
@@ -48,14 +53,9 @@ export default {
     },
   },
   mounted() {
-    this.getDatas()
-      .then(() => {
-        if (!this.productFeedIsConfigured) {
-          this.$router.push({
-            name: 'onboarding',
-          });
-        }
-      });
+    if (!this.inNeedOfConfiguration) {
+      this.getDatas();
+    }
```

Comment:
why removing the 
```
 this.$router.push({
            name: 'onboarding',
          });
```
if we open this page and the product feed is not configured yet ?

Reply:
Then we show the page with a state "need configuration, please go to the onboarding page"